### PR TITLE
new testcase to helpers_linux.go

### PR DESCRIPTION
**Pull request by tianshapjq**
**What this PR does / why we need it**:
new testcase to helpers_linux.go

### CODE REVIEW

File: pkg/kubelet/cm/helpers_linux_test.go
Comment (resouer) [inline pkg/kubelet/cm/helpers_linux_test.go]: It would be better to define a int64 `Mi` globally and use `2*Mi`, `4*Mi` etc here.

**Review by resouer — CHANGES_REQUESTED** [review]
General lgtm excepts a nits.

File: pkg/kubelet/cm/helpers_linux_test.go
Comment (resouer) [inline pkg/kubelet/cm/helpers_linux_test.go]: Add a blank line under this line to group the import path.

**Comment by resouer** [comment]
/assign @derekwaynecarr 

For approve.

File: pkg/kubelet/cm/helpers_linux_test.go
Comment (tianshapjq) [reply pkg/kubelet/cm/helpers_linux_test.go]: @resouer ok, done :)

**Comment by resouer** [comment]
Please squash

**Comment by tianshapjq** [comment]
finished~ @resouer

**Review by resouer — APPROVED** [review]
/lgtm

**Comment by resouer** [comment]
@tianshapjq I've lgtm-ed the PR but you still require another approver from sig-node to get this merge. Let's ping @kubernetes/sig-node-pr-reviews for approve.

Also, you can post this PR to sig-node slack channel to get more support :)

**Comment by tianshapjq** [comment]
/retest
@kubernetes/sig-testing-pr-reviews

**Comment by tianshapjq** [comment]
ping @kubernetes/sig-node-pr-reviews

**Comment by resouer** [comment]
ping @kubernetes/sig-node-pr-reviews, anyone approve this one? It has been years for a LGTM-ed PR stay pending ...

**Comment by tianshapjq** [comment]
@derekwaynecarr can this pr approved?

**Comment by tianshapjq** [comment]
@resouer @derekwaynecarr Hi, can this pr be merged?

**Comment by resouer** [comment]
CI should be green.

/retest

**Comment by resouer** [comment]
Please fix CI

**Comment by nikhita** [comment]
@tianshapjq looks like this PR needs a rebase :(

**Pull request closed without merging**
